{"uuid": "8501e3db-525c-4ba4-957a-0ee8c7b2e4f1", "children": ["80b496bd-7ef6-488e-aaec-c361618d39b3"], "befores": [{"name": "create_token", "status": "broken", "statusDetails": {"message": "TypeError: verfiy_http_status_code() got an unexpected keyword argument 'expect_data'\n", "trace": "  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/jabir/APIAutomation/tests/test/crud/test_create_full_booking.py\", line 32, in create_token\n    verfiy_http_status_code(response_data=response, expect_data= 200)\n"}, "start": 1712375477302, "stop": 1712375478580}], "start": 1712375477302, "stop": 1712375478638}